Documentation/ProjectFile/gml/t_name.md

```diff
@@ -1 +1,4 @@
-\ogs_missing_documentation
+Name of the geometrical set, i.e., the entirety of all geometries defined in
+this file.
+Each individual geometry additionally has an own name, which has to be unique in
+the geometrical set.
```

Comment:
- Each polyline or surface has an own name, point can be given without names.
- Good point to mention the uniqueness of the names!